### PR TITLE
[vs] Add workaround for clean up macsec ports

### PR DESCRIPTION
**Pull request by kcudnik**
@Pterosaur please address this when you will be available

### CODE REVIEW

**Comment by kcudnik** [comment]
retest vs please

File: vslib/src/MACsecManager.cpp [hunk 1]
```diff
@@ -845,7 +845,10 @@ void MACsecManager::cleanup_macsec_device() const
 
     if (!exec("/sbin/ip macsec show", macsecInfos))
     {
-        SWSS_LOG_THROW("Cannot show MACsec ports");
+        // this is workaround, there was exception thrown here, and it probably
+        // suggest that, ip command is in place, but don't support "macsec"
+        SWSS_LOG_ERROR("Cannot show MACsec ports");
```
Comment (lguohan) [inline vslib/src/MACsecManager.cpp:850]: i think this will block log analyzer, better to change to notice level.

Comment (kcudnik) [reply vslib/src/MACsecManager.cpp:850]: what log analyzer ? out internal ? do you propose warning ?

Comment (lguohan) [reply vslib/src/MACsecManager.cpp:850]: the log analyzer in sonic-mgmt repo, it will report error when it does the vsimage check, warning should be fine.

**Review by lguohan — APPROVED** [review]

**Pull request merged**